src/status_im2/contexts/chat/lightbox/animations.cljs

```diff
@@ -0,0 +1,31 @@
+(ns status-im2.contexts.chat.lightbox.animations
+  (:require [react-native.reanimated :as reanimated]))
+
+;; TODO: Abstract Reanimated methods in a better way, issue:
+;; https://github.com/status-im/status-mobile/issues/15176
+(defn get-val
```

Comment:
My personal opinion here. Having 2 vars doing the same thing only hurts maintainability. I don't think we should alias core reanimated functions just for brevity. I'm talking particularly about `get-val`, `set-val`, `use-val`, and `animate-decay`.

Except for the `animate` function, the others are doing nothing, just creating more indirection, since they're just aliases.

@OmarBasem, the code in this namespace should use the idiomatic Clojure way to alias functions, which is with `def`, but as I shared my opinion, I think this hurts maintainability, so we shouldn't do that anyway. Other developers might start to have their own ideas for how other functions should be named and a bunch of duplicated vars will start to creep up. This practice sets a bad example IMO.

I can already see with my broken crystal ball that in the future, another namespace will copy the same code used here just to use shorter animation function names in another part of the codebase.

```clojure
(def get-val reanimated/get-shared-value)
(def set-val reanimated/set-shared-value)
(def use-val reanimated/use-shared-value)
(def animate-decay reanimated/animate-shared-value-with-decay)
```

Reply:
That's not a new file, just renamed it. Also, it should be temporary. As for the aliases, will fix them.

Reply:
> That's not a new file

Oh yeah, I remember when these functions were brought to life in another PR.

Do you mean the entire file is going to be temporary? I didn't know that.

Reply:
Yes the entire file is temporary